common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -943,6 +951,17 @@ private static boolean isOsx0() {
         return osx;
     }
 
+    private static boolean isLinux0() {
+        String osname = SystemPropertyUtil.get("os.name", "").toLowerCase(Locale.US)
+                                          .replaceAll("[^a-z0-9]+", "");
+        boolean linux = osname.startsWith("linux");
```

Comment:
iirc this is not enough to detect most Linux distributions.

Reply:
```java
boolean linux = (osname.contains("nix") || osname.contains("nux")
                    || osname.contains("aix"));
```

Reply:
This would be incorrect.

 Aix != Linux, like wise picking up nix is the ending for unix again != Linux.

Epoll is Linux 2.6+ only